### PR TITLE
test: add security control checks to CI

### PR DESCRIPTION
**Pull request by ahmad-ibra**
Requires:
- https://github.com/spectrocloud/nft/pull/200

### CODE REVIEW

File: .release-please-manifest.json
Comment (ahmad-ibra) [inline .release-please-manifest.json:1]: fixing this name while im at it

**Review by TylerGillson — DISMISSED** [review]
lgtm - but we should wait for a new Bulwark release & update to the Spectro image before merging

**Pull request merged**
